lib/widgets/drive/drive-scanner.coffee

```diff
@@ -0,0 +1,27 @@
+compare = require('./compare')
+EventEmitter = require('events')
+_ = require('lodash')
+
+module.exports = class DriveScanner extends EventEmitter
+	constructor: (driveFinder,options) ->
+		super()
+		@drives = options.drives
```

Comment:
If we don't pass `options.drives` to the constructor, `@drives` will be `undefined`. We have to make sure `@drives` is an empty array in this case (also please add two test cases that assert that `@drives` is an empty array if no `options.drives`, and that it is equal to `options.drives` is this option was passed).